### ISSUE TITLE
Android-Kotlin: 02.2 Collection in Kotlin

### ISSUE THREAD

**Issue by ayush-sleeping** [girlscript/winter-of-contributing#487]

**Comment by ayush-sleeping** [comment]
I would like to work on this issue. ( Collection in Kotlin )

**Comment by sitamadex11** [comment]
yeah, you can start working.
Supervisor @rahulsain

**Comment by LavleenAgrawal** [comment]
I want to make video content on this, May I ?
Lavleen Agrawal (Batch 5 - Android Kotlin)
@sitamadex11

**Comment by ayush-sleeping** [comment]
> I want to make video content on this, May I ?
> Lavleen Agrawal (Batch 5 - Android Kotlin)
> @sitamadex11 

and I am working on documentation on this Topic

**Comment by sitamadex11** [comment]
yes

**Comment by shubhmittal07** [comment]
I would like to contribute into this topic . Can you please assign the documentation part to me.

**Comment by rahulsain** [comment]
@shubhmittal07 Doc part of this issue has already been merged

**Comment by sitamadex11** [comment]
@LavleenAgrawal Any update regarding your task?

**Comment by LavleenAgrawal** [comment]
@sitamadex11 yes sir completed...but remaining to attach on github because it isnt allowed to create multiple pull request....and i dont know how to attach it to same pull request...please guide

**Comment by sitamadex11** [comment]
@LavleenAgrawal you should make your desired changes in your branch through which you have made the PR. If you still can't resolve please ping us on discord.

**Comment by sitamadex11** [comment]
@LavleenAgrawal extremely sorry for late reply.
